metadata-ingestion/src/datahub/utilities/file_backed_collections.py

```diff
@@ -307,6 +351,17 @@ def close(self) -> None:
     def __del__(self) -> None:
         self.close()
 
+    def __enter__(self) -> "FileBackedDict":
```

Comment:
if you inherit from our `Closeable` class, you don't need to manually define enter and exit